src/modules/cpu/kernel/resize_mirror_normalize.hpp

```diff
@@ -0,0 +1,1103 @@
+                                                    Rpp32f *stdDevTensor,
+                                                    Rpp32u *mirrorTensor,
+                                                    RpptROIPtr roiTensorPtrSrc,
+                                                    RpptRoiType roiType,
+                                                    RppLayoutParams layoutParams)
+{
+    RpptROI roiDefault = {0, 0, (Rpp32s)srcDescPtr->w, (Rpp32s)srcDescPtr->h};
+
+omp_set_dynamic(0);
+#pragma omp parallel for num_threads(dstDescPtr->n)
+    for(int batchCount = 0; batchCount < dstDescPtr->n; batchCount++)
+    {
+        RpptROI roi;
+        RpptROIPtr roiPtrInput = &roiTensorPtrSrc[batchCount];
+        compute_roi_validation_host(roiPtrInput, &roi, &roiDefault, roiType);
+
+        compute_dst_size_cap_host(&dstImgSizes[batchCount], dstDescPtr);     // Check if the dstImgSize exceeds dst buffer size
+        Rpp32f wRatio = ((Rpp32f)(roi.xywhROI.roiWidth)) / ((Rpp32f)(dstImgSizes[batchCount].width));
+        Rpp32f hRatio = ((Rpp32f)(roi.xywhROI.roiHeight)) / ((Rpp32f)(dstImgSizes[batchCount].height));
+        Rpp32u heightLimit = roi.xywhROI.roiHeight - 1;
+        Rpp32u widthLimit = roi.xywhROI.roiWidth - 1;
+        Rpp32f hOffset = (hRatio - 1) * 0.5f;
+        Rpp32f wOffset = (wRatio - 1) * 0.5f;
+        Rpp32s kernelSize = 2;
+        Rpp32s noOfCoeffs = 4; // kernelSize * kernelSize
+        Rpp32s vectorIncrementPerChannel = 8;
+        Rpp32s vectorIncrementPkd = 24;
+
+        Rpp8u *srcPtrChannel, *dstPtrChannel, *srcPtrImage, *dstPtrImage;
+        srcPtrImage = srcPtr + batchCount * srcDescPtr->strides.nStride;
+        dstPtrImage = dstPtr + batchCount * dstDescPtr->strides.nStride;
+        srcPtrChannel = srcPtrImage + (roi.xywhROI.xy.y * srcDescPtr->strides.hStride) + (roi.xywhROI.xy.x * layoutParams.bufferMultiplier);
+        dstPtrChannel = dstPtrImage;
+
+        Rpp32u alignedLength = dstImgSizes[batchCount].width & ~7;   // Align dst width to process 8 dst pixels per iteration
+        __m256 pWRatio = _mm256_set1_ps(wRatio);
+        __m256 pWOffset = _mm256_set1_ps(wOffset);
+        __m256 pWidthLimit = _mm256_set1_ps((float)widthLimit);
+        __m256 pWeightParams[noOfCoeffs], pBilinearCoeffs[noOfCoeffs], pDstLoc;
+        Rpp32f weightParams[noOfCoeffs], bilinearCoeffs[noOfCoeffs];
+        Rpp32s srcLocationColumnArray[8] = {0};     // Since 8 dst pixels are processed per iteration
+        Rpp32s srcLocationRow, srcLocationColumn;
+
+        Rpp32f mean = meanTensor[batchCount];
+        Rpp32f stdDev = stdDevTensor[batchCount];
+        Rpp32f invStdDev = 1.0 / stdDev;
+        Rpp32u mirrorFlag = mirrorTensor[batchCount];
+        Rpp32u width = dstImgSizes[batchCount].width;
+
+        __m256 pRMNParams[2];
+        pRMNParams[0] = _mm256_set1_ps(mean);
+        pRMNParams[1] = _mm256_set1_ps(invStdDev);
+
+        __m256 pDstLocInit = _mm256_setr_ps(0, 1, 2, 3, 4, 5, 6, 7);
```

Comment:
Please use the predefined constant avx_pDstLocInit here

Reply:
Done